content/learn/06.hardware/nano-pcb-guide/nano-pcb-guide.md

```diff
@@ -0,0 +1,149 @@
+
+To power the Nano board using a battery, you will need to use the VIN pin (refer to the table in the section above).
+
+![Nano battery connection.](assets/nano-external-power.png)
+
+Note that Nano boards does not have a battery protection circuit, meaning it will continue to drain the battery even when it is below the discharge value, which can damage the battery. This issue can be addressed by using a battery with a built in battery protection circuit.
+
+## Pinout
+
+Nano boards largely share the placement of many pins, to make it easy for accessories to be designed for different Nano boards.
+
+### Serial Buses
+
+The Nano Family boards have serial buses attached to the following pins:
+
+| Protocol | Pins                        |
+| -------- | --------------------------- |
+| UART     | RX,TX                       |
+| SPI      | COPI(11), CIPO(12), SCK(13) |
+| I²C      | SDA(A4), SCL(A5)            |
+
+The location of these pins are located in the pinout for each board. These are found in the **Resources Section** product page of each board.
+
+## Nano Form Factor Design Files
+
+### PCB
+
+![Nano carrier template file.](assets/carrier-templates.png)
+
+Below are template files for creating your custom **Printable Circuit Board (PCB)**.
+
+- [Nano Carrier PCB template (solder pads)](/resources/cad/NanoCarrier_SolderPads.zip) - an empty carrier template with solder pads. This is useful if you want to create a design where you solder the Nano directly on top of the PCB.
+- [Nano Carrier PCB template (connectors)](/resources/cad/NanoCarrier_Socket.zip) - an empty carrier template with connectors. This is useful if you want to create a design where you can attach and remove a Nano board easily.
+
+The files can be used together with various PCB design programs, such as [Altium](https://www.altium.com/) and [Eagle](https://www.autodesk.com/products/eagle/free-download) to create your own Nano accessories.
+
+### 3D Files
+
+The design file(s) below can be used for 3D printing, e.g. enclosures, mounts.
+
+- [Nano Board Enclosure (.stl)](/resources/3d/nano-enclosure.stl)
+- [Nano Board Enclosure (.stp)](/resources/3d/nano-enclosure.stp)
+
+
+
+## Soldering Directly To PCB
+
+All Nano boards can be purchased **without headers attached.** This makes it possible to solder it directly to a custom PCB, using the castellated holes on the board. 
+
+![Castellated holes on a Nano board to the left, solder pads on a PCB to the right.](assets/castellated-pads.png)
+
+This method is useful for more robust applications, where the Nano board needs to be permanently attached.
+
+***Do not attempt to solder any Arduino Nano boards using the SMT (Surface Mount Technology) method. Since Arduino boards are shipped without anti-static bags, the board may absorb humidity which will cause issues during the SMT process. Boards should always be soldered manually.***
```

Comment:
```suggestion
***Do not attempt to solder any Arduino Nano boards using the reflow soldering method. Since Arduino boards are shipped without anti-static bags, the board may absorb humidity which will cause issues during the reflow process. Boards should always be soldered manually.***
```

I'm not certain, but I don't think "SMT method" correctly conveys the intended meaning here. Manual soldering a surface mount device would still be considered SMT soldering. So the first part of this paragraph gave me the impression that I am being told not to use the castellated pads at all.

I know what is really intended is that I should not use a reflow oven or hot air gun to solder the Nano board, while of course manual soldering the castellated pads is fine. The third sentence makes it more clear, but since I go into it confused, I might think that sentence is referring to through hole soldering using headers.

Reply:
Hey @per1234 thanks for that suggestion. I was working on addressing some other things, so I included it in another commit.